repl/lib/js/main.js

```diff
@@ -1,3 +1,6 @@
+import sanctuary from 'sanctuary';
+import ramdaFantasy from 'ramda-fantasy';
```

Comment:
Hmm, indeed, good point. Is there a CDN where these libraries are available?

A few thing come to mind
- showing a "deps up to date" badge in the repo https://david-dm.org/
- presenting the version somewhere in the REPL (It's occurred to me that it may be useful to be able to dynamically swap versions in the REPL - e.g. for people migrating between releases)